packages/health/android/src/main/kotlin/cachet/plugins/health/HealthPlugin.kt

```diff
@@ -1451,10 +1455,62 @@ class HealthPlugin(private var channel: MethodChannel? = null) :
             return
         }
 
+        getTotalInInterval(start, end, result, STEPS, AGGREGATE_STEP_COUNT, "estimated_steps", true)
+    }
+
+    private fun getTotalCaloriesInInterval(call: MethodCall, result: Result) {
+        val start = call.argument<Long>("startTime")!!
+        val end = call.argument<Long>("endTime")!!
+
+        if (useHealthConnectIfAvailable && healthConnectAvailable) {
+            getCaloriesHealthConnect(start, end, result)
+            return
+        }
+
+        getTotalInInterval(
+            start,
+            end,
+            result,
+            ACTIVE_ENERGY_BURNED,
+            AGGREGATE_CALORIES_COUNT,
+            "calories",
```

Comment:
Please consider changing `calores` to `merge_calories_expended`

Reply:
Thanks, I added this change.